src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -68,6 +68,45 @@ public static partial class Kernel32
         public unsafe delegate bool EnumResNameProc(IntPtr hModule, char* lpszType, char* lpszName, IntPtr lParam);
 
         /// <summary>
+        ///     An application-defined callback function used with the EnumResourceLanguages and EnumResourceLanguagesEx
+        ///     functions. It receives the type, name, and language of a resource item. The ENUMRESLANGPROC type defines
+        ///     a pointer to this callback function. EnumResLangProc is a placeholder for the application-defined
+        ///     function name.
+        /// </summary>
+        /// <param name="hModule">
+        ///     A handle to the module whose executable file contains the resources for which the languages are being
+        ///     enumerated.
+        ///     <para>
+        ///         If this parameter is NULL, the function enumerates the resource languages in the module used to
+        ///         create the current process.
+        ///     </para>
+        /// </param>
+        /// <param name="lpType">
+        ///     The type of resource for which the language is being enumerated. Alternately, rather than a pointer,
+        ///     this parameter can be <see cref="MAKEINTRESOURCE" />(ID), where ID is an integer value representing a predefined
+        ///     resource type. For standard resource types, see Resource Types. For more information, see the Remarks
+        ///     section below.
+        /// </param>
+        /// <param name="lpName">
+        ///     The name of the resource for which the language is being enumerated. Alternately, rather than a
+        ///     pointer, this parameter can be <see cref="MAKEINTRESOURCE" />(ID), where ID is the integer identifier
+        ///     of the resource. For more information, see the Remarks section below.
+        /// </param>
+        /// <param name="wLanguage">
+        ///     The language identifier for the resource for which the language is being enumerated. The
+        ///     EnumResourceLanguages or EnumResourceLanguagesEx function provides this value. For a list of the primary
+        ///     language identifiers and sublanguage identifiers that constitute a language identifier, see
+        ///     <see cref="MAKELANGID" />.
+        /// </param>
+        /// <param name="lParam">
+        ///     The application-defined parameter passed to the <see cref="EnumResourceLanguages(SafeLibraryHandle, char*, char*, EnumResLangProc, IntPtr)"/> EnumResourceLanguages
+        ///     or EnumResourceLanguagesEx function. This parameter can be used in error checking.
+        /// </param>
+        /// <returns>Returns TRUE to continue enumeration or FALSE to stop enumeration.</returns>
+        [UnmanagedFunctionPointer(CallingConvention.Winapi)]
+        public unsafe delegate bool EnumResLangProc([In] IntPtr hModule, [In] char* lpType, [In] char* lpName, ushort wLanguage, [In] IntPtr lParam);
```

Comment:
Can you drop the `[In]` attributes? We don't put them on any other method signatures and I don't _think_ it provides any meaningful functionality. If you know otherwise, I'd be happy to learn.